### ISSUE TITLE
FIAS is unavailable

### ISSUE THREAD

**Issue by mbarev** [liquetsoft/fias-laravel#60]
**Describe the bug**
New laravel installation (^8.4)
New fias v 9.1
Migrated successfully 
Config published successfully

Emitting full installation and getting error^

`Error while running Liquetsoft\Fias\Component\Pipeline\Task\CheckStatusTask task: FIAS is unavailable.`

**To Reproduce**
1. Install version **9.1**
2. Run command **php artisan liquetsoft:fias:install**
3. See error
```
Error while running Liquetsoft\Fias\Component\Pipeline\Task\CheckStatusTask task: FIAS is unavailable.

  at vendor/liquetsoft/fias-component/src/Pipeline/Pipe/ArrayPipe.php:142
    138▕         ]);
    139▕ 
    140▕         $this->proceedCleanup($state);
    141▕ 
  ➜ 142▕         throw new PipeException($message, 0, $e);
    143▕     }
    144▕ 
    145▕     /**
    146▕      * Обработка завершения задачи.

```

**Comment by mbarev** [comment]
Нашел проблему в самом коде

```
 // иногда версия появляется без ссылки на xml архив
            $message = "There is no xml archive set for {$versionId} complete version.";
            throw new FiasInformerException($message);

```

**Comment by marvin255** [comment]
Hello @mbarev. 

Looks like that FIAS service is unhealthy. I assume it will be ok after some time. Could you please wait for a bit and try again later?

Also you can use `php artisan liquetsoft:fias:status` to obtain current status of FIAS service.
E.g.
```
bash-5.1$ php artisan liquetsoft:fias:status 
FIAS is not available.
+-------------+---------------+------------------------------------------------------------+
| Service     | Status        | Reason                                                     |
+-------------+---------------+------------------------------------------------------------+
| informer    | not available | There is no xml archive set for 20210625 complete version. |
| file server | unknown       | Informer is unavailable                                    |
+-------------+---------------+------------------------------------------------------------+
```
